### PR TITLE
Details popup section

### PR DESCRIPTION
**Pull request by Geoelycom**
This section contains the Popup window section. I would like to add that I have refactored the whole code to fix all issues raised from code reviews from the beginning of this project.

1. Deleted the work section and recreated it with javascript dynamically
2. Re-Wrote the CSS to fit all screen sizes
3. created An Array of objects that contains all data of the work-section
4. Added code to index.html that contains the popup text
5. Added new images that are of the right sizes
6. fixed issues raised on how I implemented media queries by changing my implementation and  writing it to fit on the three main screens available
7. changed my animations from images to text and logo
8. Made use of the default export function to keep my javascript code separated

### CODE REVIEW

File: jsfiles/index.js [hunk 1]
```diff
@@ -1,16 +1,277 @@
...
+// }
+// const mobileProjects = document.querySelector('.project-works');
+// const desktopProjects = document.querySelector('.grid-container')
+
+// const projects = [mobileProjects, desktopProjects];
+// for (let i = 0; i < projects.length; i++){
+// 	console.log(projects[i]);
+// }
+
+
```
Comment (mricanho) [inline jsfiles/index.js:277]: - [ ] [OPTIONAL] I'm gonna put this as optional, but for future projects, all commented code must be gone, it's unnecesary.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Geoelycom 🔥

I'm very pleased with your project, your clean code and the array structure you followed, you are like a pro, super good job. 

I can sense all the time you put in learn and applied the knowledge, it's part of the process and it's a process that will never stop, if you conquer it, you will be a top developer.

that's how you do it

[OPTIONAL]

- You need to improve your commit messages, use a imperative mood e.g. instead of `Added delete button image and cleanup code for readibility` use `Add delete button image and clean up code for readibility`, [here is a link](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) with more information.

- You have an overflow in the x-axis, there must be a width somewhere causing the problem, [open this link](https://rawcdn.githack.com/Geoelycom/M-portfolio/5861061610c1863393f7c053a998799d2ead8cfd/index.html) on your smartphone and take note.

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Geoelycom** [comment]
Commits messages are really hard to write. Too little is bad, too much is too descriptive. I'm having a hard time really with it.

**Comment by Geoelycom** [comment]
> # Status: ✔️ Project Approved ✔️
> Hello @Geoelycom 🔥
> 
> I'm very pleased with your project, your clean code and the array structure you followed, you are like a pro, super good job.
> 
> I can sense all the time you put in learn and applied the knowledge, it's part of the process and it's a process that will never stop, if you conquer it, you will be a top developer.
> 
> that's how you do it
> 
> [OPTIONAL]
> 
> * You need to improve your commit messages, use a imperative mood e.g. instead of `Added delete button image and cleanup code for readibility` use `Add delete button image and clean up code for readibility`, [here is a link](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) with more information.
> * You have an overflow in the x-axis, there must be a width somewhere causing the problem, [open this link](https://rawcdn.githack.com/Geoelycom/M-portfolio/5861061610c1863393f7c053a998799d2ead8cfd/index.html) on your smartphone and take note.
> 
> Now you can merge this branch 🚀
> 
> ### 💻 👩🏽‍💻 Happy Coding! 👨🏽‍💻 🖥️
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

I HAVE TAKEN a look at it, I would have to try and fix this during the weekend. thank you for pointing it out.

File: jsfiles/index.js (hunk 1 above)
Comment (Geoelycom) [reply jsfiles/index.js:277]: I forgot to delete that. I used it this morning for the code review session when I was explaining the code to my morning session group.

**Pull request merged**